cql3/values.hh

```diff
@@ -42,8 +42,10 @@ struct unset_value {
 /// \brief View to a raw CQL protocol value.
 ///
 /// \see raw_value
+class raw_value;
```

Comment:
Please move this above the `raw_value_view` blurb, so the blurb stays with its class.